bears/general/QuotesBear.py

```diff
@@ -51,7 +53,8 @@ def correct_single_line_str(self, filename, file, sourcerange,
                      diff.affected_code(filename), diffs={filename: diff})
 
     def run(self, filename, file, dependency_results,
-            preferred_quotation: str='"'):
+            preferred_quotation: str='"',
+            force_preferred_quotation: bool=False):
```

Comment:
actually we already have `preferred_quotation`, what does `force_preferred_quotation` now do?

Reply:
nvm